docs/CHANGELOG.asciidoc

```diff
@@ -44,3 +44,384 @@ This section summarizes the changes in each release.
 
 //[float]
 //=== Known Issues
+
+[[release-notes-7.0.0-alpha1]]
+== {kib} 7.0.0-alpha1
+
+coming[7.0.0-alpha1]
+
+[float]
+[[breaking-7.0.0]]
+=== Breaking Changes
+
+Also see <<breaking-changes-7.0>>.
+
+Discover::
+* Does not apply `query:queryString:options` to `query_string` filters {pull}15640[#15640]
+* Removes `default_field` from `query:queryString:options` {pull}18966[#18966]
+
+Monitoring::
+* Removes `node_resolver` setting {pull}21181[#21181]
+
+Operations::
+* Removes tribe node support {pull}16397[#16397]
+* Creates separate startup scripts for development and production {pull}13806[#13806]
+* Sets default port based on protocol  {pull}21564[#21564]
+* Removes deprecated `/shorten` API {pull}21861[#21861]
+
+Platform::
+* Upgrades Hapi in legacy platform to v17 {pull}21707[#21707]
```

Comment:
This can be removed. The breaking change here was really just for plugins, which we don't flag in this way.